app/src/main/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -37,7 +39,14 @@ class MessagingService : FirebaseMessagingService() {
         const val TAG = "MessagingService"
         const val TITLE = "title"
         const val MESSAGE = "message"
+        const val SUBJECT = "subject"
+        const val IMPORTANCE = "importance"
+        const val TIMEOUT = "timeout"
         const val IMAGE_URL = "image"
+        const val ICON = "icon"
```

Comment:
I think that `icon` might be a confusing option name for users, since the frontend has an icon set and here it means a url. 

I do understand that it's confusing, as we also already have `IMAGE_URL`. 

Maybe we should update it to be `ICON_URL` so at least it's clear it's a url ?

Reply:
I agree with @balloob, once this is fixed I think the PR looks good!

Reply:
Yes i agree also. But the image property is also called ``image`` without the ``_url``. Maybe the property should also be renamed to ``image_url`` because the property also takes only URLs? Right now just the var is called ``image_url``